scripts/raceday_verify.py

```
@@ -343,16 +343,27 @@ def check_rt_data_freshness(con, year, monthday, issues, stale_minutes=15):
+    # Check TS_O1 freshness — the best live signal (updates every ~60s during races)
+    latest_ts = q(con, "SELECT MAX(HassoTime) FROM TS_O1 WHERE Year=? AND MonthDay=?", (y, m))
+    ts_snaps  = q(con, "SELECT COUNT(DISTINCT HassoTime) FROM TS_O1 WHERE Year=? AND MonthDay=?", (y, m)) or 0
+    if latest_ts:
+        # HassoTime format MMDDHHMM: extract HH:MM for display
+        h_str = str(latest_ts)
+        hhmm = f"{h_str[4:6]}:{h_str[6:8]}" if len(h_str) >= 8 else h_str
+        print(f"  [OK]  TS_O1 latest snapshot: {hhmm}  ({ts_snaps} snapshots today)")
+    else:
+        now_h = datetime.now().hour
+        if now_h >= 10:
+            print(f"  [!]   TS_O1 no snapshots yet after 10:00 (realtime odds not flowing)")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)